src/library_html5_webgpu.js

```diff
@@ -1,11 +1,34 @@
 mergeInto(LibraryManager.library, {
   // TODO(kainino0x): make it possible to actually create devices through webgpu.h
   emscripten_webgpu_get_device__deps: ['$WebGPU'],
-  emscripten_webgpu_get_device__postset: 'WebGPU.initManagers();',
   emscripten_webgpu_get_device: function() {
 #if ASSERTIONS
     assert(Module['preinitializedWebGPUDevice']);
 #endif
     return WebGPU["mgrDevice"].create(Module['preinitializedWebGPUDevice']);
   },
+
```

Comment:
Actually -   this import/export stuff isn't even needed. If we put all the WebGPU objects in the same namespace, a small code snippet like:

```
class JsHandle {
  public:
    JsHandle(struct EmJsHandle* handle) : mHandle(handle){};
    ~JsHandle() { emscripten_unwrap_js_handle(mHandle); }

    struct EmJsHandle* Get() { return mHandle; }

  private:
    struct EmJsHandle* mHandle;
};

template <typename T> JsHandle export_js_handle(const T& object) {
  return JsHandle(EM_ASM_JS_VAL({ return WebGPU['allObjects'].get($0); }, object.Get()));
}
```

is pretty sufficient which we can probably just provide in the html5_webgpu.h header.

Reply:
The WebGPU opaque objects are all `int`s conjured up just for the `mgr`s anyway, can we just completely replace the `mgr`s with JsValStore?

Reply:
Probably, but let's look at that in the future.

Reply:
Wasn't sure if it would be easier to skip the intermediate step, since it sounds kind of complicated. Either way SGTM.

Reply:
I think we should probably think carefully about whether a single object store would cause us to run out of ids too quickly for a heavy app. At the very least, we can definitely factor stuff to share code though.

Reply:
Since the IDs wrap around, don't we have 2^31 currently-live ones to work with?